Site/SiteAnalyticsModule.php

```diff
@@ -213,7 +242,126 @@ public function getGoogleAnalyticsTrackerInlineJavascript()
 
 			$javascript = sprintf(
 				$javascript,
-				$this->getTrackingCodeSource()
+				$this->getGoogleAnalyticsTrackingCodeSource()
+			);
+		}
+
+		return $javascript;
+	}
+
+	// }}}
+	// {{{ public function hasGoogleAnalytics()
+
+	public function hasFacebookPixel()
+	{
+		return (
+			$this->facebook_pixel_id != '' &&
+			$this->analytics_opt_out === false
+		);
+	}
+
+	// }}}
+	// {{{ public function pushFacebookPixelCommand()
+
+	public function pushFacebookPixelCommand($command)
+	{
+		$this->facebook_pixel_commands = array_merge(
```

Comment:
`array_merge()` will merge arrays, not push new values. Do you mean to use `array_push()` or should you use the `[] =` operator?